rts/lra/lra-test/pom.xml

```diff
@@ -28,7 +28,7 @@
 
     <artifactId>lra-test</artifactId>
     <packaging>jar</packaging>
-    <name>Run the example locally</name>
+    <name>LRA Quickstart lra-test locally</name>
```

Comment:
Just to mention, IMO cleanup is good and should be done - I do think it should be in a separate commit though (same comment for the change below in rts/lra/pom.xml) /cc @ochaloup @mayankkunwar